docs/examples/3rdparty_analysis_workflow.sh

```diff
@@ -4,8 +4,11 @@ set -e
 
 # BOILERPLATE
 
-BOBS_HOME=$(readlink -f $(mktemp -d datalad_demo.XXXX))
-ALICES_HOME=$(readlink -f $(mktemp -d datalad_demo.XXXX))
+BOBS_HOME=$(readlink -f "$(mktemp --tmpdir -d datalad_demo_bob.XXXX)")
+ALICES_HOME=$(readlink -f "$(mktemp --tmpdir -d datalad_demo_alice.XXXX)")
+
+# Fake an SSH server on this machine for the purpose of this demo
+SERVER_URL=localhost:$(readlink -f "$(mktemp --tmpdir -u -d datalad_demo_testpub.XXXX)")
```

Comment:
What if there's nothing called `localhost` in my local configuration? Example doesn't work then? ;-)